include/userobjects/MyTRIMRasterizer.h

```diff
@@ -58,13 +58,14 @@ class MyTRIMRasterizer : public ElementUserObject
   struct AveragedData
   {
     AveragedData(unsigned int nvars = 0)
-      : _elements(nvars, 0.0), _Z(nvars, 0.0), _M(nvars, 0.0), _site_volume(0.0)
+      : _elements(nvars, 0.0), _Z(nvars, 0.0), _M(nvars, 0.0), _M_tol(nvars, 0.5), _site_volume(0.0)
     {
     }
 
     std::vector<Real> _elements;
     std::vector<Real> _Z;
     std::vector<Real> _M;
+    std::vector<Real> _M_tol;
```

Comment:
This dos not seem right. `AveragedData` is stored in a map with an entry for _every_ active element. I don't think we'll need spatially varying mass tolerance.

Reply:
The same is should then be true for `_M` and `_Z`.

Reply:
Yes... I think you're right!